server/market/market.go

```diff
@@ -238,7 +240,8 @@ ordersLoop:
 			// maybe a revoke failed.
 			log.Errorf("Not rebooking order %v with amount (%v/%v) incompatible with current lot size (%v)",
 				lo.FillAmt, lo.Quantity, mktInfo.LotSize)
-			if _, _, err = storage.RevokeOrder(lo); err != nil {
+			// Revoke the order, but do not count this against the user.
+			if _, _, err = storage.RevokeOrderUncounted(lo); err != nil {
```

Comment:
These three order revocations are in the part of the Market constructor that reloads the order book.  Since the market was not running, and may have changed configuration, I think it is excusable to have these forgiven if user did something to make loading these orders impossible thus triggering their revocation, e.g. spending the funding coins or somehow having an order in a different market that locks these coins. We may wish to make this more strict in the future however, since the suspend messages will indicate if the book is going to persist, although the reconfiguration parameters and the scheduled down time are out of the user's hands and thus it is hard to blame them.